src/azure-cli/azure/cli/command_modules/serviceconnector/_resource_config.py

```diff
@@ -1083,6 +1083,15 @@ class CLIENT_TYPE(Enum):
             CLIENT_TYPE.Dotnet,
             CLIENT_TYPE.Blank
         ],
+        RESOURCE.SignalR: [
+            CLIENT_TYPE.Dotnet,
+            CLIENT_TYPE.Blank
+        ],
+        RESOURCE.WebPubSub: [
+            CLIENT_TYPE.Java,
+            CLIENT_TYPE.Dotnet,
+            CLIENT_TYPE.Blank
+        ],
```

Comment:
@xfz11 please confirm the clientType here

Reply:
WebPubSub also includes Python, Nodejs

Reply:
Python, Nodejs are alreay added for webapp+webpubsub

this is for Spring-Cloud, spring-cloud itself doesn't support Python NodeJs, right